config/pipelines/detect_burnin.pipe

```diff
@@ -75,12 +75,23 @@ process detect_burnin__dilater
   filter:vxl_morphology:kernel_radius   = 2.0
 
 # =============================================================================
-process detect_burnin__out
+process detect_burnin__convert
   :: image_filter
   filter:type                           = vxl_convert_image
   filter:vxl_convert_image:type         = byte
   filter:vxl_convert_image:scale_factor = 255
 
+# =============================================================================
+process detect_burnin__merge
+  :: merge_image
+  merge_images:type                     = ocv
+
+# =============================================================================
+process detect_burnin__out
```

Comment:
I think maybe we should call this `inpaint` rather than `out`.  There may be multiple "outs" if the user of this file wants to save both masks and inpainted results.